pkg/apis/pingcap/v1alpha1/tikv_config.go

```diff
@@ -742,3 +744,72 @@ type TiKVCoprocessorConfig struct {
 	// optional
 	RegionSplitKeys *int64 `json:"region-split-keys,omitempty" toml:"region-split-keys,omitempty"`
 }
+
+type TiKVEncryptionConfig struct {
```

Comment:
add `// +k8s:openapi-gen=true` to generate open api spec of these structs

Reply:
done